File: CrowdInvestWebApp/src/app/models/InvestmentFund.ts
import { InvestmentFundContribution } from "./InvestmentFundContribution";


export class InvestmentFund {
    investmentFundId: string;
    fundTitle: string;
    fundDescription: string;
    maximumValue: number;
    currentTotal: number;
    contribution: InvestmentFundContribution;
}

File: CrowdInvestCore/Models/InvestmentFund.cs
using System;

namespace CrowdInvestCore.Models
{
	public class InvestmentFund
	{
		public InvestmentFund()
		{
			InvestmentFundId = Guid.NewGuid();
		}

		public Guid InvestmentFundId { get; set; }

		public string FundTitle { get; set; }

		public string FundDescription {get; set; }

		public decimal MaximumValue { get; set; }
	}
}